lib/proxy/ip-forward.php

```diff
@@ -14,15 +14,15 @@
  *
  * @return (bool) true, if REMOTE_ADDR updated; false, if not.
  */
-function fix_remote_address( $user_ip, $remote_proxy_ip, $proxy_ip_whitelist ) {
+function fix_remote_address( $user_ip, $proxy_ip_whitelist ) {
 	if ( ! is_valid_ip( $user_ip ) ) {
 		return false;
 	}
 
 	require_once( __DIR__ . '/ip-utils.php' );
 
 	// Verify that the remote proxy matches our whitelist
-	$is_whitelisted_proxy_ip = IpUtils::checkIp( $remote_proxy_ip, $proxy_ip_whitelist );
+	$is_whitelisted_proxy_ip = IpUtils::checkIp( $_SERVER['HTTP_X_FORWARDED_FOR'], $proxy_ip_whitelist );
```

Comment:
Minor: do we need to verify that this SERVER var is set before accessing it (to prevent a notice)?